deployment/terraform/coordinator.go

```diff
@@ -77,6 +79,36 @@ func (t *Terraform) StartCoordinator() error {
 		return fmt.Errorf("error running ssh command: output: %s, error: %w", out, err)
 	}
 
+	// Uploading simul controller config
+	simulConfig, err := simulcontroller.ReadConfig("")
+	if err != nil {
+		return err
+	}
+	data, err = json.MarshalIndent(simulConfig, "", "  ")
+	if err != nil {
+		return err
+	}
+	mlog.Info("Uploading simulcontroller config file")
+	dstPath = "/home/ubuntu/mattermost-load-test-ng/config/simulcontroller.json"
+	if out, err := sshc.Upload(bytes.NewReader(data), dstPath, false); err != nil {
+		return fmt.Errorf("error running ssh command: output: %s, error: %w", out, err)
+	}
+
+	// Uploading simple controller config
+	simpleConfig, err := simplecontroller.ReadConfig("")
+	if err != nil {
+		return err
+	}
+	data, err = json.MarshalIndent(simpleConfig, "", "  ")
+	if err != nil {
+		return err
+	}
+	mlog.Info("Uploading simplecontroller config file")
+	dstPath = "/home/ubuntu/mattermost-load-test-ng/config/simplecontroller.json"
+	if out, err := sshc.Upload(bytes.NewReader(data), dstPath, false); err != nil {
+		return fmt.Errorf("error running ssh command: output: %s, error: %w", out, err)
+	}
```

Comment:
Can this be compressed somehow? We do the same things, just the controller type changes.

Reply:
Sure, let me take look.

Reply:
better now ?

Reply:
Yes, wondering why you didn't use`uploadBatch()` though.

Reply:
Because I needed to encode the JSON too. So it would be two loops. 0/5.

Reply:
Fair enough :+1: 
We can probably pass `bytes.NewReader(data)` directly since it's pretty clear what it returns.